content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -149,6 +155,56 @@ the possible following fields:
 
 `dvc.yaml` files also support `# comments`.
 
+> Note that unlike `.dvc` files, `dvc.yaml` doesn't store any `md5` hash values,
+> making it extra easy to edit manually. These are kept in `dvc.lock` instead.
+
+### dvc.lock file
+
+The purpose of `dvc.lock` is to track the latest version of all files involved
```

Comment:
track -> pin, capture (check docs for Gemfile.lock, or yarn.lock, etc)

Reply:
it also plays a role of `.dvc` file for pipeline ouputs.

I guess we need to be more explicit here.

Reply:
> pin, capture (check docs for Gemfile.lock, or yarn.lock

How about "lock"? 🙂  I checked https://docs.npmjs.com/files/package-lock.json and came up with a new intro. PTAL

Reply:
> plays a role of .dvc file for pipeline ouputs

Should I add a note to connect these 2 concepts? Per https://github.com/iterative/dvc.org/pull/1448#pullrequestreview-433572182 I don't think that's necessary.